core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -528,25 +527,23 @@ var ExplorationEditorMainTab = function() {
   // ---- HINT ----
 
   this.addHint = async function(hint) {
-    await addHintButton.click();
+    await action.click('Add hint button', addHintButton);
     var addHintModal = element(
       by.cssContainingText('.protractor-test-hint-modal', 'Add Hint'));
     await waitFor.visibilityOf(
       addHintModal, 'Add hint modal takes too long to appear');
-    await element(by.css('.protractor-test-hint-text')).all(by.tagName('p'))
-      .last().click();
+    var hintButton = await element(
+      by.css('.protractor-test-hint-text')).all(by.tagName('p')).last();
+    await action.click('Hint Button', hintButton);
     await browser.switchTo().activeElement().sendKeys(hint);
```

Comment:
Do we really have to send keys this way instead of `action.sendKeys`? If so, do we need an extra `await` somewhere in here?